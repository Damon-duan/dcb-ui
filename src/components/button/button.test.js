import { shallowMount } from '@vue/test-utils';
import Button from './button.vue';

describe('Button', () => {
  test('渲染正常', () => {
    const wrapper = shallowMount(Button);
    expect(wrapper.html()).toMatchSnapshot();
  });
});
