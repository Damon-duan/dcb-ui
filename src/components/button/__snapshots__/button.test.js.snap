// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button 渲染正常 1`] = `<div>按钮</div>`;
